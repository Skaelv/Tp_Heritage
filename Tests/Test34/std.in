C C1 10 13 24
C C2 1 1 1
C C3 1 1 1
UNDO
UNDO
MOVE C1 1 1
REDO
EXIT
