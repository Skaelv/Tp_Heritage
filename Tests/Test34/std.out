 # Entrez une commande
OK
# New Object : C1
 # Entrez une commande
OK
# New Object : C2
 # Entrez une commande
OK
# New Object : C3
 # Entrez une commande
OK
 # Entrez une commande
OK
 # Entrez une commande
OK
# C1 has been moved
 # Entrez une commande
# No command to execute
 # Entrez une commande

